src/main/java/org/osiam/ResourceServer.java

```diff
@@ -67,13 +55,16 @@
     @Value("${org.osiam.resource-server.db.vendor}")
     private String databaseVendor;
 
-    @Bean
-    public static PropertySourcesPlaceholderConfigurer properties() {
-        PropertySourcesPlaceholderConfigurer propertySources = new PropertySourcesPlaceholderConfigurer();
-        Resource[] resources = new ClassPathResource[] { new ClassPathResource("resource-server.properties") };
-        propertySources.setLocations(resources);
-        propertySources.setIgnoreUnresolvablePlaceholders(true);
-        return propertySources;
+    public static void main(String[] args) {
+        System.setProperty("spring.jpa.hibernate.naming_strategy", "org.hibernate.cfg.ImprovedNamingStrategy");
```

Comment:
This needs to be set because the default naming strategy is set by Spring to [`SpringNamingStrategy`](https://github.com/spring-projects/spring-boot/blob/master/spring-boot/src/main/java/org/springframework/boot/orm/jpa/hibernate/SpringNamingStrategy.java) which just extends `ImprovedNamingStrategy`. The following error message occurred when using `SpringNamingStrategy`:

```
org.hibernate.AnnotationException: Unable to create unique key constraint (user_internal_id, extension_field)
on table scim_extension_field_value: database column 'extension_field' not found. Make sure that you use
the correct column name which depends on the naming strategy in use (it may not be the same as the
property name in the entity, especially for relational types)
```

But the column `extension_field` exists...